### PR TITLE
Cleanup + led overlay expansion

### PR DESCRIPTION
**Pull request by ASDosjani**
Betaflight changes: https://github.com/betaflight/betaflight/pull/12554

Reverts #3391 because that doesn't solve the problem on my machine.
Please test if it looks the same for you.

Master:
image
My PR:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* approving -- looks symmetrical light and dark themed.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
